File: app/src/debug/java/com/soumya/wwdablu/zomatobuddy/BuddyDebugTree.java
package com.soumya.wwdablu.zomatobuddy;

import timber.log.Timber;

public class BuddyDebugTree extends Timber.DebugTree {

    @Override
    protected String createStackElementTag(StackTraceElement element) {
        return super.createStackElementTag(element) + "[" + element.getLineNumber() + "]";
    }
}

File: app/src/debug/java/com/soumya/wwdablu/zomatobuddy/BuddyDebugTree.kt
package com.soumya.wwdablu.zomatobuddy

import timber.log.Timber.DebugTree

class BuddyDebugTree : DebugTree() {
    override fun createStackElementTag(element: StackTraceElement): String {
        return super.createStackElementTag(element) + "[" + element.lineNumber + "]"
    }
}